src/Sylius/Bundle/AdminBundle/Message/CreateAdminUser.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Message;
```

Comment:
```suggestion
namespace Sylius\Bundle\AdminBundle\Command;
```

Reply:
IIRC some time ago it was agreed upon using the `Message*` namespaces to not conflate CQRS with Symfony's CLI commands